src/test/java/com/rocketden/main/api/RoomTests.java

```diff
@@ -242,7 +242,8 @@ public void createAndJoinRoom() throws Exception {
 
     @Test
     public void createAndJoinRoomUserAlreadyExists() throws Exception {
-        // POST request to create room and PUT request to join room should succeed
+        // POST request to create room and PUT request to join room should fail
+        // because the room already has a user with the exact same information
```

Comment:
Multi-line comments usually have different styling than the single line `//` comments.
```suggestion
        /**
         * POST request to create room and PUT request to join room should fail
         * because the room already has a user with the exact same information
         */
```

Reply:
Do you mind also changing the comments for `createAndJoinRoomNoUser`? That's the last nit, just updating the ones in that method so they actually reflect what's going on (I know this wasn't your method in the first place, so thanks for fixing that lol).